src/main/java/eu/yals/constants/Header.java

```diff
@@ -13,6 +13,7 @@ private Header() {
 
     public static final String LOCATION = "Location";
     public static final String CONTENT_TYPE = "Content-Type";
+    public static final String CONTENT_LENGTH = "Content-Length";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/kyberorg/yals/pullRequest?prid=5150905)